R/serialization.R

```diff
@@ -0,0 +1,42 @@
+# This file will be autogenerated.
+
+#' Serialize/Unserialize an mlpack model.
+#'
+#' @title Mlpack-Serialization.
```

Comment:
I'm not sure what the title here means?

Reply:
`@title` appears in the inline help file. I think this is a remanent from the suggestion to merge help docs. Only `@rdname` needs to be set. 

FYI: Quick illustration for how the roxygen2 params work.

![image](https://user-images.githubusercontent.com/833642/85236480-02da5580-b3e4-11ea-9e88-5597ebcfc560.png)

@Yashwants19 you may wish to either delete this `@title Mlpack-Serialization` or remove the prior lines with:

```r
#' Serialize/Unserialize an mlpack model.
#'
```

Reply:
Deleted `@title Mlpack-Serialization`.